docs/modules/ROOT/pages/guide-metatx.adoc

```diff
@@ -0,0 +1,349 @@
+
+When comparing the code, note the metatransaction's use of `_msgSender()` as opposed to the SimpleRegistry's use of `msg.sender`. By extending from `ERC2771Context` and `MinimalForwarder`, the contract becomes metatransaction capable. (Note that all OpenZeppelin contracts are compatable with the use of `_msgSender()`.)
+
+The second fundamental change between the two contracts is the need for the metatransaction contract to specify the address of the trusted forwarder, which in this case is the address of the `MinimalForwarder` contract.
+
+[[configure-project]]
+== Configure the project
+
+First, fork the repository and then `git clone` your fork to your computer and install dependencies
+
+```
+$ git clone https://github.com/[GitHub username]/workshops.git
+$ cd workshops/25-defender-metatx-api/
+$ yarn
+```
+
+Create a `.env` file in the project root and supply your Team API key and secret from Defender. A private key will be used for local testing but the Relayer is used for actual contract deployment.
+
+```
+PRIVATE_KEY="Goerli private key"
+TEAM_API_KEY="Defender Team API key, used for uploading autotask code"
+TEAM_API_SECRET="Defender Team API secret"
+```
+
+[[create-relayer]]
+== Create Relayer
+
+First, install the necessary package:
+
+`npm install defender-relay-client`
+
+Create a script such as the following, supplying your Defender API key in your .env file and adjusting variable names as appropriate.
+
+```
+const { RelayClient } = require('defender-relay-client');
+const { appendFile, readFileSync, writeFileSync} = require('fs');
+
+async function run() {
+  require('dotenv').config();
+  const { TEAM_API_KEY: apiKey, TEAM_API_SECRET: apiSecret, AUTOTASK_ID: autotaskId } = process.env;
+  const relayClient = new RelayClient({ apiKey, apiSecret });
+
+  // create relay using defender client
+  const requestParams = {
+    name: 'MetaTxRelayer',
+    network: 'goerli',
+    minBalance: BigInt(1e17).toString(),
+  };
+  const relayer = await relayClient.create(requestParams);
+  
+  // store relayer info in file
+  writeFileSync('relay.json', JSON.stringify({
+    relayer
+  }, null, 2));
+  console.log('Relayer ID: ', relayer);
+
+  // create and save the api key to .env - needed for sending tx
+  const {relay} = JSON.parse(readFileSync('relay.json'))
+  const secretKey = await relayClient.createKey(relayer.relayerId);
+  writeFileSync('.env.relay', JSON.stringify({secretKey}, null, 2));
+}
+
+run().catch((error) => {
+  console.error(error);
+  process.exitCode = 1;
+});
+```
+
+Run with `yarn create-relay`
```

Comment:
Let's clarify that this can also be done in the UI by manually creating a relay, and link to the docs